tests/test_attributes.py

```diff
@@ -327,3 +362,24 @@ def test_uuid(token, client, Uuid):
         client=client,
         token=token,
     )
+
+def test_part_table(token, client, ParentPart):
+    ParentPart[0].insert1(dict(scan_id=0, data=5))
+    ParentPart[1].populate()
```

Comment:
Curious to try this first.
```suggestion
def test_part_table(token, client, **ParentPart):
    ScanData.insert1(dict(scan_id=0, data=5))
    ProcessScanData.populate()
```
If not, can at least do the following to make it easier to follow:
```suggestion
def test_part_table(token, client, ParentPart):
    ScanData, ProcessScanData = ParentPart
    ScanData.insert1(dict(scan_id=0, data=5))
    ProcessScanData.populate()
```

Reply:
Fair point, was being lazy

Reply:
Went with the dict approach